style.css

```diff
@@ -199,6 +199,35 @@ input{
 /*    border-right: 1px dotted#D8D8D8; */
 }
 
+/* details[open] summary::marker {
+color: red;
+}
+
+details:not([open]) summary::marker {
+color: royalblue;
+} */
+
+summary::marker, summary::-webkit-details-marker {
+  display: none;
+}
+
+summary:after {
+  /* background: red; */
+  /* border-radius: 5px; */
+  content: "+";
+  color: #000;
+  float: left;
+  font-size: 1.5em;
+  font-weight: bold;
+  margin: -5px 10px 0 0;
```

Comment:
```suggestion
  margin: -10px 10px 0 0;
```

Suggesting this adjustment as it helped alignment. Went from:
![image](https://user-images.githubusercontent.com/5214594/98185346-d599ca00-1ee2-11eb-8ea1-88d7ce916ed5.png)
to
![image](https://user-images.githubusercontent.com/5214594/98185397-eb0ef400-1ee2-11eb-95c4-82ea280c0e6a.png)